conf/core/Horizontal.conf

```diff
@@ -0,0 +1,5 @@
+{
+  "conflate.pre.ops": "$(conflate.pre.ops);hoot::CookieCutterOp",
+  "cookie.cutter.alpha": "2500",
+  "#": "end"
+}
```

Comment:
@maxgrossman this file shouldn't be added, it exists in `conf/core/HorizontalConflation.conf`